### ISSUE TITLE
Angular async pipe return type is 'string|null' not 'string' as [highlight] expect.

### ISSUE THREAD

**Issue by konradkurylo** [MurhafSousli/ngx-highlightjs#209]
#### Reproduction

Since Angular async pipe return type is 'string|null', the code suggested in your README
```
<pre>
  <code [highlight]=" 'https://my.magic.url/snippet' | codeFromUrl | async"></code>
</pre> 
```

is not compiling and display error below:

Type 'string | null' is not assignable to type 'string'.

Temporarily ugly solution:

```
<pre>
  <code [highlight]=" ('https://my.magic.url/snippet' | codeFromUrl | async) ?? ' ' "></code>
</pre> 
```

Target solution: 

(1) Write custom angular pipe to assign default value, and use it after async pipe. I have suggested orElseEmpyString name. It will be convenient to attach this pipe to your library.

```
<pre>
  <code [highlight]=" 'https://my.magil.url/snippet' | codeFromUrl | async | orElseEmptyString"></code>
</pre> 
```

(2) Make [higlight] variable assignable to 'string|null' type, and implement proper behavior to handle that fact.

**Comment by MurhafSousli** [comment]
Please add reproduction here https://stackblitz.com/edit/ngx-highlightjs!  You seem to have strict type template on your end.. I don't think there is a need for another pipe

**Comment by jasomimo** [comment]
Hi @MurhafSousli, I experienced same issue as @konradkurylo with `"strictTemplates": true` option in angularCompilerOptions, tsconfig.json.

I don't think there is a need to for a new pipe, but the `highlight` directive should already handle if the value is `null`, which is possible return value from async pipe.

In the `ngOnChanges` of [highlight.ts](https://github.com/MurhafSousli/ngx-highlightjs/blob/master/projects/ngx-highlightjs/src/lib/highlight.ts#L54), there is already check for `currentValue !== null`.

I believe `@Input('highlight') code!: string;` could be changed to `@Input('highlight') code!: string | null;` and that should fix the error and hopefully don't break anything.

I like you package and I'm happy to open PR, if you would agree with the change.

**Comment by MurhafSousli** [comment]
@jasomimo In this case, you need to remove the bang from the variable because it means the variable will not be null, as explained here

https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci

Try to verify if this fixes the issue or if other variables need to be updated the same way!

Thanks a lot!

**Comment by jasomimo** [comment]
@MurhafSousli I've updated `highlight.ts` to accept also `null` as valid value. I saw there is already a test for `null` value, so I didn't add a new one. Change is available for a review in PR: #225

I tested ngx-highlightjs with this change in my project, where I was previously getting error and the fix worked well. 

Looking forward to the feedback, if this works for you.

**Comment by MurhafSousli** [comment]
Sorry for the late, v6.1.3 has been published!
